### PR TITLE
Tried to change this... but needs possibly major refactoring

### PR DESCRIPTION
**Pull request by JJ**
###### Motivation

Trying to fix moving average on the origin, where votes are tallied. I'm afraid it needs also some fixes on the other end.  Relates to #226 

###### Changes

- [x] Ensured that you have [rebased your branch](https://stackoverflow.com/a/7244456) with this repo's latest master branch.
- [x] Ensured that relevant issues are linked, if this PR resolves any outstanding.
- [ ] Added a screenshot for all UI changes (you can drag the file into this edit box and it will be uploaded).
- [ ] Ensured that changes to auto-generated files have not been committed; in particular, `*.html`, `*.csv` and `*.json` files.

### CODE REVIEW

File: print-battleground-state-changes [hunk 1]
```diff
@@ -261,17 +261,21 @@ def json_to_summary(
     candidate2 = row.candidates[1] # Trailing candidate
     candidate1_name = candidate1['last_name']
     candidate2_name = candidate2['last_name']
-    candidate1_votes = candidate1['votes']
-    candidate2_votes = candidate2['votes']
-    vote_diff = candidate1_votes - candidate2_votes
+    votes_c = {}
+    votes_c[candidate1_name] = candidate1['votes']
+    votes_c[candidate2_name] = candidate2['votes']
+    if votes_c[candidate1_name] > votes_c[candidate1_name]:
```
Comment (chuckie79) [inline print-battleground-state-changes]: Should the RHS of this if statement be candidate2_name?

Comment (JJ) [reply print-battleground-state-changes]: votes_c now contains the number of votes...

Comment (chuckie79) [reply print-battleground-state-changes]: Sorry, I will reword this question;

Should the RHS side of this if statement be "votes_c[candidate2_name]" ?

At the moment the number of votes for candidate1 is being compared to itself.

**Comment by spraot** [comment]
This is already fixed by #215

File: print-battleground-state-changes (hunk 1 above)
Comment (JJ) [reply print-battleground-state-changes]: Ah, OK, I see what you mean, it's the last line. Anyway, I'm closing this. I've reviewed other PR and looks much more promising.

**Pull request closed without merging**